app/components/Dashboard/MarketsTable.jsx

```diff
@@ -242,10 +243,12 @@ class MarketsTable extends React.Component {
             <div>
                 <div className="header-selector">
                     <div className="filter inline-block">
-                        <input
+                        <Input
```

Comment:
Please don't break alignemnt

before
![image](https://user-images.githubusercontent.com/33128181/49018049-a755d080-f18a-11e8-8eb9-b3d49f4be7e4.png)

after
![image](https://user-images.githubusercontent.com/33128181/49018039-a1f88600-f18a-11e8-9b44-e0d13fd5508d.png)

Reply:
not clear. which file you refer to ? Do  not see "Active | Hidden "  onwallet.bitshares.com